manifests/params.pp

```diff
@@ -133,27 +133,23 @@
     }
   }
 
-  if fact('aio_agent_version') =~ String[1] {
```

Comment:
If you drop this, you should also drop the tests around it. I realize it's a draft, but making  a note here so we don't forget.